src/pages/settings/Profile/ProfilePage.js

```diff
@@ -66,56 +65,29 @@ class ProfilePage extends Component {
     constructor(props) {
         super(props);
 
-        this.defaultAvatar = ReportUtils.getDefaultAvatar(this.props.currentUserPersonalDetails.login);
-
+        this.defaultAvatar = ReportUtils.getDefaultAvatar(this.props.myPersonalDetails.login);
```

Comment:
`myPersonalDetails` was renamed to `currentUserPersonalDetails`. Can you merge main and make sure that nothing broke with that change?